.github/workflows/test.yml

```diff
@@ -1,28 +1,30 @@
+---
 name: Test
+permissions: read-all
 
 on: [push, pull_request]
 
 jobs:
-  test_java8_node_12:
-    name: Test - Debian - Java 8 - Node 12
+  test_java8_node_16:
+    name: Test - Debian - Java 8 - Node 16
     runs-on: ubuntu-latest
     container:
-      image: openjdk:8-jdk-stretch
+      image: eclipse-temurin:8
```

Comment:
The `openjdk:8-jdk-stretch` image was causing this job to fail on the "Install apt packages" step. See: https://github.com/tylervz/npm-groovy-lint/actions/runs/4949858376/jobs/8852637916

For posterity, here is some of the log output since eventually GitHub will delete the log file from that GH Action run which failed:
```text
Err:10 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
  404  Not Found
Ign:11 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Err:9 http://deb.debian.org/debian stretch/main amd64 Packages
  404  Not Found
Err:11 http://deb.debian.org/debian stretch-updates/main amd64 Packages
  404  Not Found
Ign:12 http://deb.debian.org/debian stretch-updates/main all Packages
Reading package lists...
W: The repository 'http://security.debian.org/debian-security stretch/updates Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch-updates Release' does not have a Release file.
E: Failed to fetch http://security.debian.org/debian-security/dists/stretch/updates/main/binary-amd64/Packages  404  Not Found
E: Failed to fetch http://deb.debian.org/debian/dists/stretch/main/binary-amd64/Packages  404  Not Found
E: Failed to fetch http://deb.debian.org/debian/dists/stretch-updates/main/binary-amd64/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
Error: Process completed with exit code 100.
```